enterprise_subsidy/apps/transaction/signals/handlers.py

```diff
@@ -28,6 +30,14 @@ def listen_for_transaction_reversal(sender, **kwargs):
         raise ValueError(msg)
     try:
         EnterpriseApiClient().cancel_fulfillment(transaction.fulfillment_identifier)
+        subsidy_redemption = SubsidyRedemption(
+            subsidy_identifier=transaction.subsidy_access_policy_uuid,
```

Comment:
Hmm, doesn't have to be addressed in this PR, but `subsidy_identifier` is a really confusing name for this key - as a casual reader of the code, I'd expect a key with that name to point to an instance of `subsidy.models.Subsidy` :)

Reply:
@iloveagent57 the challenge was to come up with a reasonably generic identifier without injecting too much of our policy/budget/subsidy lingo into the event.